docs/elastic.md

```diff
@@ -36,8 +41,7 @@ spec:
   test_user: homer_simpson
   clustername: test_ci
   elasticsearch:
-    server: my.es.server
-    port: 8080
+    url: "http://my.es.server:9200"
```

Comment:
Can we give an example with a user/pass being passed as well?

Reply:
done!